pfrl/experiments/train_agent_async.py

```diff
@@ -209,11 +211,15 @@ def train_agent_async(
             If set to None, a new Event object is created and used internally.
 
     Returns:
-        Trained agent.
+        agent: Trained agent.
+        eval_stats_history: Not supported for train_agent_async. Always returns None.
     """
 
     logger = logger or logging.getLogger(__name__)
 
+    if len(evaluation_hooks) > 0:
+        assert RuntimeError("evaluation_hooks is not supported for train_agent_async.")
```

Comment:
I'm not confident of this file:
I modified this file to unify the interface of `train_agent_async` and `train_agent/train_agent_batch`.

Reply:
fair enough, I have reverted the changes.